docs/classic/03-cti-adapter/01-cti-adapter-configuration.md

```diff
@@ -93,6 +93,8 @@ will need to create the Amazon Connect Single Sign On URL and validate
 that it works correctly, then configure the Lightning CTI adapter and
 login the agent.
 
+> **Note: With the new Amazon Connect instance urls (`*.my.connect.aws`) you must put the full URL into the `Amazon Connect Instance` field in the AC CTI Adapter record for SSO to work. Ex: using `https://myinstance.my.connect.aws` instead of `my instance`.**
```

Comment:
i don't believe this is unique to the sso use case - if their instance uses the new url schemes then they must specify the whole url sso or not

Reply:
Oh interesting, I've been able to have it embed without sso for the new instances. It gives me a sameorigin error, but it still lets me embed it.